Cargo.lock

```diff
@@ -1977,11 +1977,12 @@ dependencies = [
  "anyhow",
  "clap 3.0.0-beta.2",
  "near-indexer",
+ "near-o11y",
  "openssl-probe",
  "serde_json",
  "tokio",
  "tracing",
- "tracing-subscriber",
+ "tracing-subscriber 0.2.25",
```

Comment:
Let’s try to dedup these

Reply:
tracing-span-tree would need an update... Not entirely sure what the motivation was behind maintaining our own.

Reply:
Publishes the new version of tracing span tree!  It’s its own thing because I wasn’t able to find an existing subscriber which gives an output, useful for quick profiling.

now that we wave o11y crate in near, it might be a good idea to just copy the contents here: that’s a tiny amount of code, it isn’t worth it to keep a separate crate for that.